R/annotate_r_version.R

```diff
@@ -0,0 +1,29 @@
+#' Annotate R Version
+#'
+#' @importFrom rstudioapi getActiveDocumentContext insertText versionInfo
+#' @importFrom utils sessionInfo
+#'
```

Comment:
we should mention in the docs to be careful using this function in markdown files to avoid breaking the yaml and preventing proper rendering